Haskell-Generate/GenRustJets.hs

```diff
@@ -234,6 +236,24 @@ rustJetDecode mod =
                      <-> ("1" <+> "=>" <+> braces (docTree r))
                        ) <> line
 
+rustJetCost :: Module -> Doc a
+rustJetCost mod = vsep $
+  [ nest 4 (vsep ("fn cost(&self) -> Cost {" :
+    [ nest 4 (vsep ("match self {" :
+        map (<>comma)
+        [ pretty modname <> "::" <> pretty (jetName jet) <+> "=>" <+>
+          if BitcoinModule == jetModule jet
+          then "unimplemented!(\"Unspecified cost of Bitcoin jets\")"
```

Comment:
In b4d9fcdea97eb293db03ccd8ea4f57071dc8b86c:

We should move the `unimplemented!` up to the entire function instead of doing the match and then repeating it a zillion times.

Reply:
We do the same for Bitcoin jet pointers, but I can update both.

Reply:
Updating both sounds good, yeah.

Reply:
Done